MediaPicker/src/main/java/org/wordpress/android/mediapicker/MediaPickerActionModeCallback.kt

```diff
@@ -1,102 +0,0 @@
-package org.wordpress.android.mediapicker
-
-import android.view.Menu
-import android.view.MenuItem
-import android.view.View
-import android.widget.TextView
-import androidx.appcompat.view.ActionMode
-import androidx.appcompat.view.ActionMode.Callback
-import androidx.appcompat.widget.TooltipCompat
-import androidx.lifecycle.Lifecycle
-import androidx.lifecycle.Lifecycle.Event.ON_START
-import androidx.lifecycle.Lifecycle.Event.ON_STOP
-import androidx.lifecycle.LifecycleOwner
-import androidx.lifecycle.LifecycleRegistry
-import androidx.lifecycle.Observer
-import org.wordpress.android.R
-import org.wordpress.android.mediapicker.MediaPickerViewModel.ActionModeUiModel
-import org.wordpress.android.ui.utils.UiString.UiStringRes
-import org.wordpress.android.ui.utils.UiString.UiStringText
-
-class MediaPickerActionModeCallback(private val viewModel: MediaPickerViewModel) : Callback,
```

Comment:
🔧  I think this class is used in `MediaPickerFragment` : https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/pull/33/files#diff-7f84ecd9a646ff8eb7edd2717f7a4d5b4347da3be0ec1851d321b1573a25b9ebR253

Maybe we shouldn't remove it, WDYT?

Reply:
That's a good point! Android studio didn't really show any usages so I just removed it after looking at the class and the fact that it focuses on menu options like editing, which we don't support at the moment. 

But I'll add it back to this PR and if I find it's not needed in `MediaPickerFragment` I'll remove it again.